news.html

```diff
@@ -62,99 +62,91 @@ <h1 id="logo"><a href="index.html">Celestia</a></h1>
-										<hr />
-										<h3>Celestia 1.6.0</h3>
-                                        The next major update of Celestia is finally ready. Download version 1.6.0 for Mac, Windows, and Linux from SourceForge. 1.6.0 adds a lot of new capabilities to Celestia. <a href="/features/160-features.html">This list</a> describes a few of them. For a thorough list of what has changed, have a look at the <a href="http://sourceforge.net/project/shownotes.php?group_id=21302&release_id=695491" target="_blank">change log</a>.
-										<hr />
-										<h3>Celestia 1.5.1</h3>
-                                        Celestia version 1.5.1 is now available for download from SourceForge. 1.5.1 is a bug fix and translation release that offers no new features beyond what it is in 1.5.0. The translations in 1.5.1 are much more complete than in 1.5.0 and fonts with Japanese, Chinese, and Cyrillic character sets are now included in the standard package. See the short <a href="/151changes.txt">change list</a> for details on the bug fixes. 
-										<hr />
-										<h3>Celestia 1.5.0</h3>
-                                        After nearly two years of development, the next major version is finally ready. You can download the Mac OS X package, Windows package, and source tarball from SourceForge. Read the <a href="/features/150-features.html">1.5.0 Feature Summary</a> to find out what's new in this version. 
-										<hr />
-										<h3>New Celestia Web Site</h3>
-                                        Welcome to the newly redesigned Celestia web site. Thanks are due to Runar Thorvaldsen, who is responsible for the new and vastly improved look of the site. I apologize for the long down time and hope that you enjoy the new layout. 
-										<hr />
-	
-									</section>
-
-							</div>
-						</div>
+<table>
+  <tr>
+    <td class="icon fa-calendar">Nov 19, 2020</td>								    <td class="8u$ 12u$(medium) important(medium)">
+      <h3>Celestia 1.6.2</h3>
+      <p>Version 1.6.2 of Celestia is now available for all supported systems&nbsp;&mdash; see the <a href="/download.html">download page</a>. 1.6.2 contains bug fixes, new and more complete translations, and supports <em>64-bit</em> Windows and macOS systems. This is especially important for macOS Catalina and newer versions, which have dropped support for 32-bit apps. This version also supports brown dwarves class Y backported from version 1.7. There are no data updates in this release; they will be added later, but for now you can use <a href="/forum/viewtopic.php?f=23&t=20533" target="_blank">Celestia Origin</a> and/or other addons <a href="/forum/viewtopic.php?f=23&t=20205" target="_blank">from the Celestia Forums</a>.</p>
```

Comment:
I'd suggest removing mention of support for class Y and instead linking to the changelog, since it's now been added to the repository.